ansible/playbooks/Supporting_Scripts/Nagios_Ansible_Config_tool/Nagios_Ansible_Config_tool.sh

```diff
@@ -131,16 +117,18 @@ if [ ! -f $Nagios_Logo_Folder/$Sys_Icon_Picked.gd2 ] ; then													# If the
 	echo "Logo icon was not found" $Sys_Icon_Picked "Defaulting to nagios icon"
 	Sys_Icon_Picked=nagios.gd2
 fi
-###################
-# Package Manager #
-###################
-# Detect the right package manager to monitor
+#############################
+# Distro Specific Templates #
+#############################
+# Detect the right package manager to monitor and which script to check the network timesync
 Sys_OS="yes"
 case "$Distro" in
-        Ubuntu)
-                Sys_OS_pkg_Template=$Template_Dir/apt.cfg ;;
+        Ubuntu|Debian)
+                Sys_OS_pkg_Template=$Template_Dir/apt.cfg
+                Sys_OS_timesync_Template=$Template_Dir/check_timesync.cfg ;;
         RedHat|CentOS)
-                Sys_OS_pkg_Template=$Template_Dir/yum.cfg ;;
+                Sys_OS_pkg_Template=$Template_Dir/yum.cfg
+                Sys_OS_timesync_Template=$Template_Dir/check_ntp_timesync.cfg ;;
         SLES)
```

Comment:
No time check on other platforms?

Reply:
I checked SLES, and that can use the `check_timesync` plugin so I've added that in, Looking at the single FreeBSD machine we have, it doesn't have `timedatectl` or `ntpstat`, so neither will work for the moment :/

Reply:
If we recreate old (non-systemd) systems, we should think about using chrony. Chrony is available on a wide range of platforms.

Reply:
Agreed. As it currently stands, we don't do anything regarding system time on FreeBSD, in the playbooks, so we could add a `check_chrony` script at the same time :-)